### PR TITLE
the format of src-url in sd-local

### PR DESCRIPTION
**Pull request by s-yoshika**
## Context
We implemented the `src-url` option in https://github.com/screwdriver-cd/sd-local/pull/13, and the format of the url is not mentioned by the design docs.

## Objective
This adds the details of `src-url` format.

## References
https://github.com/screwdriver-cd/sd-local/pull/13
https://github.com/screwdriver-cd/screwdriver/issues/1314

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: design/sd-local.md [hunk 1]
```diff
@@ -136,6 +137,12 @@ $ sdlocal build [job-name] [options]
 - `-m, --memory [size]` Set memory size which Build Container can use. Either b, k, m, g can be used as a size unit. (default: ?)
 - `--src-url [repository url]` Set repository URL which is to build when user use the remote repository without local files.
 
+###### src-url option
+- How to specify the url
```
Comment (tk3fftk) [inline design/sd-local.md]: There are "the url" and "the URL".

File: design/sd-local.md [hunk 2]
```diff
@@ -136,6 +137,12 @@ $ sdlocal build [job-name] [options]
...
+  - The URL can be passed with ether https or ssh schema
```
Comment (tk3fftk) [inline design/sd-local.md]: ether -> either

File: design/sd-local.md (hunk 1 above)
Comment (s-yoshika) [reply design/sd-local.md]: fixed

File: design/sd-local.md (hunk 2 above)
Comment (s-yoshika) [reply design/sd-local.md]: thanks, I fixed.

**Pull request merged**